### DISCUSSION TITLE
ipynb Jupyter Notebook outputed figures have scrollbars

### DISCUSSION THREAD

**Discussion by bugsbunny88** [quarto-dev/quarto-cli#5270] [Q&A]
I am extremely confused,
I've tried fig-size options and fig-responsive to no avail.
what ends up happening to every generated plot are scrollbars within the site html that is generated
ive also changed options in the library utilized. im not sure whats controlling that behavior or how to investigate.

**Comment by mcanouil** [comment]
Could you share a complete and small Quarto document (this includes Jupyter Notebook given to Quarto) exhibiting your issue that we can render?
Also what version of Quarto are you using and what operating system are you on?
